get_all_blobs/getallblobs.js

```diff
@@ -0,0 +1,181 @@
+            "name": "l2LogsTreeRoot",
+            "type": "bytes32"
+          },
+          {
+            "internalType": "uint256",
+            "name": "timestamp",
+            "type": "uint256"
+          },
+          {
+            "internalType": "bytes32",
+            "name": "commitment",
+            "type": "bytes32"
+          }
+        ],
+        "internalType": "struct IExecutor.StoredBatchInfo",
+        "name": "",
+        "type": "tuple"
+      },
+      {
+        "components": [
+          {
+            "internalType": "uint64",
+            "name": "batchNumber",
+            "type": "uint64"
+          },
+          {
+            "internalType": "uint64",
+            "name": "timestamp",
+            "type": "uint64"
+          },
+          {
+            "internalType": "uint64",
+            "name": "indexRepeatedStorageChanges",
+            "type": "uint64"
+          },
+          {
+            "internalType": "bytes32",
+            "name": "newStateRoot",
+            "type": "bytes32"
+          },
+          {
+            "internalType": "uint256",
+            "name": "numberOfLayer1Txs",
+            "type": "uint256"
+          },
+          {
+            "internalType": "bytes32",
+            "name": "priorityOperationsHash",
+            "type": "bytes32"
+          },
+          {
+            "internalType": "bytes32",
+            "name": "bootloaderHeapInitialContentsHash",
+            "type": "bytes32"
+          },
+          {
+            "internalType": "bytes32",
+            "name": "eventsQueueStateHash",
+            "type": "bytes32"
+          },
+          {
+            "internalType": "bytes",
+            "name": "systemLogs",
+            "type": "bytes"
+          },
+          {
+            "internalType": "bytes",
+            "name": "pubdataCommitments",
+            "type": "bytes"
+          }
+        ],
+        "internalType": "struct IExecutor.CommitBatchInfo[]",
+        "name": "_newBatchesData",
+        "type": "tuple[]"
+      }
+    ],
+    "name": "commitBatchesSharedBridge",
+    "outputs": [],
+    "stateMutability": "nonpayable",
+    "type": "function"
+  }
+]
+
+const contract = new web3.eth.Contract(abi);
+
+function hexToUtf8(hex) {
+  // Remove the '0x' prefix if present
+  if (hex.startsWith('0x')) {
+    hex = hex.slice(2);
+  }
+
+  // Ensure the hex string has an even length
+  if (hex.length % 2 !== 0) {
+    throw new Error('Invalid hex string length');
+  }
+
+  // Convert hex string to a byte array
+  const bytes = [];
+  for (let i = 0; i < hex.length; i += 2) {
+    bytes.push(parseInt(hex.substr(i, 2), 16));
+  }
+
+  // Convert byte array to UTF-8 string
+  const utf8String = new TextDecoder('utf-8').decode(new Uint8Array(bytes));
+  return utf8String;
+}
+
+async function getTransactions() {
+  const latestBlock = await web3.eth.getBlockNumber();
+  const validatorTimelockAddress = "0xeacf0411de906bdd8f2576692486383797d06004";
```

Comment:
please parameterize the `validatorTimelockAddress` and the `commitBatchesSharedBridge_functionSelector`

Reply:
Done!